its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SonarQubeTestSuite.java

```diff
@@ -45,6 +45,7 @@ public class SonarQubeTestSuite {
     .addPlugin(FileLocation.of(TestUtils.getCustomRoslynPlugin().toFile()))
     .addPlugin(TestUtils.getMavenLocation("org.sonarsource.dotnet", "sonar-csharp-plugin", System.getProperty("sonar.csharpplugin.version", "DEV")))
     .addPlugin(TestUtils.getMavenLocation("org.sonarsource.dotnet", "sonar-vbnet-plugin", System.getProperty("sonar.vbnetplugin.version", "DEV")))
+    .addPlugin(TestUtils.getMavenLocation("org.sonarsource.javascript", "sonar-javascript-plugin", System.getProperty("sonar.javascriptplugin.version", "7.4.4.15624")))
```

Comment:
Maybe mention this line in[ this](https://github.com/SonarSource/sonar-scanner-msbuild/issues/1486) issue and say why the version is hardcoded, and we can actually use dev once that issue is fixed.